pkgs/development/python-modules/tinygrad/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "tinygrad";
+  version = "0.7.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "tinygrad";
+    repo = "tinygrad";
+    rev = "v${version}";
+    hash = "sha256-lXpMqG8yNYkYZ0LZh5ERYCViELTauwiijDRAmQZIfQI=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+    wheel
+  ];
+
+  pythonImportsCheck = [ "tinygrad" ];
+
+  meta = with lib; {
+    description = "You like pytorch? You like micrograd? You love tinygrad";
```

Comment:
Although this is an upstream issue, I don't think it explains much.
IMO, the title of [this page](https://tinygrad.org/), `A simple and powerful neural network framework`, would be more appropriate.